apps/console/src/auth.html

```diff
@@ -385,13 +438,81 @@
                             sessionStorage.setItem("auth_callback_url_console", authCallbackUrl);
                         }
 
-                        auth.signIn(getAuthParams({}));
+                        if (isCentralDeploymentEnabled) {
+                            if (authorizationCode) {
+                                auth.signIn({callOnlyOnRedirect: true})
+                                    .then((response) => {
+                                        auth.getDecodedIDToken().then((token) => {
+                                            let loginTenant = token["default_tenant"];
+                                            const associatedOrgJson = token["org_user_associations"]?.join(",");
+                                            const orgUserAssociations = associatedOrgJson ? JSON.parse(associatedOrgJson) : undefined;
+                                            const requestedTenant = localStorage.getItem("user_requested_tenant");
+
+                                            localStorage.removeItem("user_requested_tenant");
+
+                                            let loginTenantAssociation;
+
+                                            if (requestedTenant) {
+                                                loginTenantAssociation = orgUserAssociations?.find((org) => org.orgHandle === requestedTenant);
+                                                if (loginTenantAssociation) {
+                                                    loginTenant = requestedTenant;
+                                                }
+                                            } else {
+                                                loginTenantAssociation = orgUserAssociations?.find((org) => org.orgHandle === loginTenant);
+                                            }
+
+                                            if (!loginTenantAssociation && orgUserAssociations?.length) {
+                                                loginTenantAssociation = orgUserAssociations[0];
+                                            }
+
+                                            if (!loginTenantAssociation) {
+                                                loginTenant = superTenantGlobal;
+                                            }
+                                            
+                                            var authConfigRegion = {
+                                                signInRedirectURL: deploymentUnitSignInRedirectURL(loginTenant),
+                                                signOutRedirectURL: getSignOutRedirectURL(),
+                                                clientID: "<%= htmlWebpackPlugin.options.clientID %>",
+                                                baseUrl: getDeploymentUnitApiPath(loginTenant),
+                                                responseMode: "query",
+                                                scope: ["openid SYSTEM profile"],
```

Comment:
is it intentional to pass this as a space separated string. IINM, we need to pass scopes as an array.

Reply:
This is how we passed currently as well: https://github.com/wso2/identity-apps/blob/dc8434613e0f2432b96efee392d3e7913c03465e/apps/console/src/auth.html#L343